src/map/clif.c

```diff
@@ -683,8 +683,10 @@ void clif_authok(struct map_session_data *sd)
 	const int cmd = 0x73;
 #elif PACKETVER < 20141022
 	const int cmd = 0x2eb;
-#else
+#elif PACKETVER < 20161228
```

Comment:
better turn the upper one into a or condition:
`#elif PACKETVER < 20141022 || PACKETVER >= 20161228`
We have it like that for other packets as well. Character deletion for example.